docs/matrix.md

```diff
@@ -1,8 +1,10 @@
-<!--
----
+## <!--
+
```

Comment:
are the doc changes in this file due to some sort of auto-formatting? Ideally this PR would only include doc changes relevant to the functional changes.

Reply:
Yes, not sure why it auto-formats or how to prevent it from doing so. I asked in the tekton-squad if there was a way to run a command to format `.md` files and Yongxuan said it was fine if it reformats the entire file.

Reply:
This doesn't look like formatting? Can we double check and make sure the doc layout is not changed by this PR? I was thinking about trimming spaces when you asked that question.